### DISCUSSION TITLE
A32nx and VRinsight MPanel

### DISCUSSION THREAD

**Discussion by Peter-Mary-Jo** [flybywiresim/aircraft#5356] [Q&A]
Please forgive me if this point has been already assessed. 
Mpanel works perfectly with ASOBO's A320 neo. It works partially with A32nx developper: Hdg and speed doesn't work. Any reason for that ?

All the best,

Pierre

**Comment by Inaxair** [comment]
Hello,

can you provide more details? 

Also, please ensure that you have moved aside all the other mods to ensure it's not a conflict of compatibility, and that you are on the latest version of the simulator.

Best regards,
Inaxair

**Reply by Peter-Mary-Jo** [reply]
Thank you for your answer.
I don't run any mod except A32nx Development, FSUIPC7 and Navigraph. My configuration is lower than requested by MSFS and I get a warning each time I launch the simulator (Intel i3-7100 @3.9GHz, GeForce GTX1050Ti 4GbVRAM, RAM 32Gb DDR4). But MSFS run OK with A32nx except for Autoland at detailed airport.
VRinsight MPanel is connected to MSFS via VRiSim.exe and works OK with ASOBO's A320neo. Most of MPanel functions works with A32nx ( frequencies setting, Altitude , Gear and Flaps,...) except that heading bug and FCU speed can't be set: looks like Mpanel can read but not write the MSFS setting for these two parameter.
Sorry for this long post.
Best regards

Pierre

**Reply by eleenokia** [reply]
Hello, I also have a VRinsight panel which with the stable version 0.6.1 worked perfect. But with the latest MSFS U5, now some keys don't work, like landing lights, taxi lights. But I think it must be something from the Sim because I can't make them work with the mouse either. What surprises me is that I do not see that someone has published something similar.
Anyway, with the DEV version, my VRinsight for several months does not work the speed and direction controls with the DEV versions of flybywire either. I thought it was just me, but I see that I'm not alone.
NOTE: With ASOBO's A320 neo version, everything works fine.

**Reply by Peter-Mary-Jo** [reply]
Hello, Nice to know that we are at least  two with the same issue with MPanel and A32nx Development. I'll give a try to MPanel +stable version 0.6.1. 
Thank you for the information
